scripts/circle-ci/run-circle-tests.sh

```diff
@@ -73,8 +75,11 @@ JAVA_GC_LOGGING_OPTIONS="${JAVA_GC_LOGGING_OPTIONS} -Xlog:class+unload=off"
 JAVA_GC_LOGGING_OPTIONS="${JAVA_GC_LOGGING_OPTIONS} -Xlog:gc:build-%t-%p.gc.log"
 
 # External builds have a 16gb limit.
-if [ "$test_suite_index" -eq "13" ]; then
+if [ "$test_suite_index" -eq "14" ]; then
     export _JAVA_OPTIONS="-Xms2g -Xmx4g -XX:ActiveProcessorCount=8 ${JAVA_GC_LOGGING_OPTIONS}"
+elif [ "$test_suite_index" -eq "3" ]; then
+    export _JAVA_OPTIONS="-Xms8g -Xmx8g -XX:ActiveProcessorCount=8 ${JAVA_GC_LOGGING_OPTIONS}"
+    BASE_GRADLE_ARGS+=" --scan --parallel"
 else
     export _JAVA_OPTIONS="-Xmx4g ${JAVA_GC_LOGGING_OPTIONS}"
```

Comment:
not needed now: It's a bit weird that we don't set Xms=Xmx across the containers. I don't know if it could make the other tests do slightly better if they don't need to resize the heap

Reply:
thanks, I'll see if that improves things!